service-base/src/main/java/fi/nls/oskari/service/ServiceException.java

```diff
@@ -1,15 +1,23 @@
 package fi.nls.oskari.service;
 
+import org.json.JSONObject;
+
 public class ServiceException extends Exception {
-    
-	private static final long serialVersionUID = -3303389634376753825L;
+    private JSONObject options;
```

Comment:
This change feels a bit much. Maybe instead add an `UserLayerException` that extends ServiceException?